handlers.js

```diff
@@ -29,7 +29,7 @@ let signerPrivateKey
 let signerWallet
 
 // A maximum amount to send. Could be determined dynamically by balance
-const maxSignerParam = 1000
+const maxSignerParam = BigNumber(100 * 10 ** constants.decimals.DAI).toString()
```

Comment:
`BigNumber(100 * 10).pow(constants.decimals.DAI).toString()`

Reply:
The `pow` only applies to the `10` which would be a separate BigNumber.

`BigNumber(100).multipliedBy(BigNumber(10).pow(constants.decimals.DAI)).toString()`

Is the internal `BigNumber(10)` necessary?

Reply:
that looks good to me. I believe the internal `BigNumber(10)` is necessary to get access to `.pow`, and in any case it shouldn't hurt.

Reply:
Sorry meant whether the `BigNumber(10).pow(n)` was necessary versus `10 ** n` but will go with the BigNumber just to be sure.